x-pack/plugins/ingest_manager/public/plugin.ts

```diff
@@ -39,7 +39,7 @@ export interface IngestManagerSetup {}
  */
 export interface IngestManagerStart {
   registerPackageConfigComponent: typeof registerPackageConfigComponent;
-  success: Promise<true>;
+  waitForInitialization: () => Promise<true>;
```

Comment:
I'm 👍 on going from `Promise<true>` to `() => Promise<true>` but would prefer a different function name like `isReady` or `isInitialized`